app/src/main/res/layout/user_map_activity.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".UserMapActivity">
+    <fragment xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
And here it says we could replace fragment with FragmentContainerView, and that the namespace declaration is redundant